src/main/java/Main.java

```diff
@@ -1,6 +1,141 @@
+import java.util.Scanner;
 
 public class Main {
+    private static int peoples;
+    private static final Goods goods = new Goods();
+    private static boolean isExit = false;
```

Comment:
```suggestion
    private static boolean isExit = false;
    private static Scanner scanner = new Scanner(System.in);
```
Лучше использовать один scanner на этот весь проект, не нужно у сканера ничего сбрасывать

Reply:
Исправил